.travis.yml

```diff
@@ -54,12 +61,12 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then source terryfy/test_tools.sh; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then python -m pytest rotkehlchen; fi
   - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then coverage run --source rotkehlchen/ -m pytest --travis-fold=always $TEST_TYPE; fi
-  - npm config set python python2.7
+  - npm config set python python3.6
```

Comment:
There is no need to change this. The reason I set npm's python here is only for electron rebuild. This has nothing to do with the python used in the tests.